optaplanner-core/src/main/java/org/optaplanner/core/api/solver/SolverJob.java

```diff
@@ -72,4 +73,14 @@
      */
     Solution_ getFinalBestSolution() throws InterruptedException, ExecutionException;
 
+    /**
+     * Returns the {@link Duration} spent solving since the last start.
+     * If it hasn't started it yet, it returns {@link Duration#ZERO}.
+     * If it hasn't ended yet, it returns the time between the last start and now.
+     * If it has ended already, it returns the time between the last start and the ending.
+     *
+     * @return the {@link Duration} spent solving since the last (re)start, at least 0
+     */
+    Duration getTimeSpent();
```

Comment:
New API. Technically, this is a break of bwd compatibility for the interface.
Partically, no way to avoid it.

Reply:
Maybe we should rename this method to getDurationSpent() or better yet, `getSolvingDuration()` Wdyt? Let's discuss on zulip.

Reply:
agreed on getSolvingDuration()